src/govuk/components/pagination/_index.scss

```diff
@@ -0,0 +1,243 @@
+@include govuk-exports("govuk/component/pagination") {
+  // Flexbox enhancement for small screen visual design
+  // Falls back to a float: left layout on non-flex browsers
+  .govuk-pagination {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    flex-wrap: wrap;
+    margin-bottom: govuk-spacing(4);
+
+    @include govuk-media-query($from: tablet) {
+      flex-direction: row;
+      align-items: flex-start;
+    }
+  }
+
+  .govuk-pagination__list {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+  }
+
+  .govuk-pagination__item,
+  .govuk-pagination__next,
+  .govuk-pagination__prev {
+    @include govuk-font(19);
+    box-sizing: border-box;
+    position: relative;
+    min-width: 45px;
+    min-height: 45px;
+    padding: govuk-spacing(2) govuk-spacing(3);
+    float: left; // Float is ignored if flex is active for prev/next links
+
+    &:hover {
+      background-color: govuk-colour("light-grey", $legacy: "grey-4");
+    }
+  }
+
+  .govuk-pagination__item {
+    // Hide items on small screens except the prev/next items,
+    // non-link items and the first and last items
+    display: none;
+
+    // Center align pagination links in their parent list item so that they
+    // visually sit in the middle of their touch area
+    text-align: center;
+
+    @include govuk-media-query($from: tablet) {
+      display: block;
+    }
+  }
+
+  .govuk-pagination__prev,
+  .govuk-pagination__next {
+    @include govuk-typography-weight-bold;
+    flex-basis: 100%;
+
+    @include govuk-media-query($from: tablet) {
+      flex-basis: auto;
+    }
+
+    // Use flex to get around a whitespace issue between the arrow svg and the link text
+    // without having to rely on whitespace control from backend tooling
+    .govuk-pagination__link {
+      display: flex;
+    }
```

Comment:
Probably one for @christopherthomasdesign or @Ciandelle to weigh in on, but to my eyes the arrows on mobile are sitting too low relative to the text:

![Screenshot 2022-05-20 at 11 17 21](https://user-images.githubusercontent.com/121939/169507715-f5c068cd-e82c-427f-bd3d-3d3fca8b651e.png)

I think removing the 5px top margin from the icon (only on mobile?) and possibly adding `align-items: baseline` to `govuk-pagination__link` gets us closer to something that looks right:

![Screenshot 2022-05-20 at 11 19 58](https://user-images.githubusercontent.com/121939/169507916-a5195e78-2741-46f0-a101-4518adc68976.png)

Reply:
Agree with @36degrees - the arrows look of centre on the first image.

Reply:
I've gone with removing the top margin altogether and using `align-items: center` as to my eye it centres it a little better.
<img width="122" alt="Screenshot 2022-05-20 at 15 38 48" src="https://user-images.githubusercontent.com/64783893/169551743-ea73ea44-db8e-4433-a038-959bda3a8b03.png">